shared/test/ui_test_credentials.json

```diff
@@ -1,5 +1,13 @@
-{
-    "username" : "__INSERT__USERNAME__HERE__",
-    "password" : "__INSERT__PASSWORD__HERE__",
-    "url" : "test.salesforce.com"
-}
\ No newline at end of file
+[
```

Comment:
Can we make the sample a standard JSON structure as well?  This will fail basic JSON parsing, and limits your ability to make assertions about the default file as a result.